docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+# Personal Access Token
+
+The API ML allows generating, validating and invalidating a **Personal Access Token (PAT)** that can be used, for instance, to leverage modern tools such as Version Control Systems (VCS) during
```

Comment:
That could help access the tools such as VCS without using the credentials of the specific person and storing the credentials in the automation.

Reply:
I would also mention here the possibility to limit access to specific services.